test/libsolidity/syntaxTests/types/weird_sized_types.sol

```diff
@@ -0,0 +1,41 @@
+contract C {
+    bytes0 b1;
```

Comment:
Unfortunately the `DeclarationError` is fatal, so if we want to test these different combinations, that means ton of files.

Reply:
This would already be an error in develop right now. At least we should add the test from the issue.

Reply:
@hrkrshnn the question here is what to do with tests.

Reply:
I don't think we need to have a test for each prefix. We know that parsing the number is not implemented individuall for each prefix.